ui/components/app/ledger-instruction-field/ledger-instruction-field.js

```diff
@@ -201,7 +201,7 @@ export default function LedgerInstructionField({ showDataInstruction }) {
               usingWebHID &&
                 webHidConnectedStatus ===
                   WEBHID_CONNECTED_STATUSES.NOT_CONNECTED,
-              COLORS.SECONDARY_DEFAULT,
+              COLORS.WARNING_DEFAULT,
             )}
           </div>
         </Dialog>
```

Comment:
These colors have the same hex value( #F66A0A ) but updating because SECONDARY_DEFAULT is deprecated

<img width="331" alt="Screen Shot 2022-08-11 at 4 13 32 PM" src="https://user-images.githubusercontent.com/8112138/184257614-567b91ce-40a3-4a87-845b-884d1d99d6b6.png"><img width="325" alt="Screen Shot 2022-08-11 at 4 13 39 PM" src="https://user-images.githubusercontent.com/8112138/184257615-01990564-1b48-4b98-aa84-4d14e430c7dc.png">